src/components/avatar/Avatar.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import * as React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
I think we've been tending toward the `*` import style and referencing things directly off the `React` alias.

Reply:
Hmm, would that mean React.useState and React.useEffect? Personally, I prefer it without the React identifier. I forget, but is there a reason we do `import * as React` instead of just `import React`? They seem equivalent based on this file (https://github.com/facebook/react/blob/master/packages/react/src/React.js)

Reply:
`import * as React` is to bring in all the Flow types